feature-group-definitions/array-flat.yml

```diff
@@ -1,5 +1,5 @@
 name: Array flat() and flatMap()
-description: Array flat() and flatMap() methods
+description: The `flat()` and `flatMap()` methods for arrays and typed arrays TODO.
```

Comment:
Promote leads me to think that the input is modified in some way, but this always creates a new array, right?

Reply:
Excellent point. Having another go at this:

```suggestion
description: The `flat()` and `flatMap()` methods for arrays and typed arrays returns a copy of an array such that the items of nested arrays become direct members of the previously enclosing array. In other words, the methods reduce the number of dimensions of a multidimensional array.
```

Reply:
I think that "become direct members of the previously enclosing array" still sounds like it's going to modify the input array. The "creates a new array"  in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat is key I think.

Reply:
Round 3:

```suggestion
description: The `flat()` and `flatMap()` methods for arrays and typed arrays creates a new array such that each nested array item is concatenated into it.
```

(I didn't steal the MDN description wholesale because I think the recursive bit is more confusing than descriptive in the default case of depth 1.)